src/pudl/metadata/fields.py

```diff
@@ -4176,6 +4176,50 @@
             "pattern": r"^\d{4}$",
         },
     },
+    "design_wind_speed_mph": {
+        "type": "number",
+        "description": "Design average annual wind speed for turbines at this generator.",
+    },
+    "obstacle_num_faa": {
+        "type": "string",
+        "description": "The FAA obstacle number assigned to this generator. If more than one obstacle number exists, the one that best represents the turbines. This field was only reported for a few years.",
```

Comment:
good point with the FAA (i copied and pasted from EIA heh). yea i didn't want to specify the years because it would be very easy for EIA to update add this info in an udpate and for this for go out of date. But i can add it :-/

Reply:
hmm good point about the years